src/box/lua/iproto.c

```diff
@@ -25,43 +45,7 @@ struct iproto_constant {
 	const lua_Integer val;
 };
 
-/**
- * Pushes an array of IPROTO constants onto Lua stack.
- */
-static void
-push_iproto_constant_subnamespace(struct lua_State *L, const char *subnamespace,
-				  const struct iproto_constant *constants,
-				  int constants_len)
-{
-	lua_createtable(L, 0, constants_len);
-	for (int i = 0; i < constants_len; ++i) {
-		lua_pushinteger(L, constants[i].val);
-		lua_setfield(L, -2, constants[i].name);
-	}
-	lua_setfield(L, -2, subnamespace);
-}
-
-/**
- * Pushes IPROTO constants related to `IPROTO_FLAG` key onto Lua stack.
- */
-static void
-push_iproto_flag_constants(struct lua_State *L)
-{
-	const struct iproto_constant flags[] = {
-		{"COMMIT", IPROTO_FLAG_COMMIT},
-		{"WAIT_SYNC", IPROTO_FLAG_WAIT_SYNC},
-		{"WAIT_ACK",  IPROTO_FLAG_WAIT_ACK},
-	};
-	push_iproto_constant_subnamespace(L, "flag", flags, lengthof(flags));
-}
-
-/**
- * Pushes IPROTO constants from `iproto_key` enumeration onto Lua stack.
- */
-static void
-push_iproto_key_enum(struct lua_State *L)
-{
-	const struct iproto_constant keys[] = {
+static const struct iproto_constant iproto_keys[] = {
```

Comment:
Bad indentation below (two tabs instead of one). Let's leave it where it was. It's okay to define a static variable in a function body.

Reply:
Fixed, moved `iproto_keys` and `iproto_key_aliases` to `push_iproto_key_enum`.